src/update/lib/cron.js

```diff
@@ -1,47 +1,70 @@
-/* eslint-disable no-restricted-syntax */
 const { CronJob } = require('cron');
-const fs = require('fs-extra');
-const path = require('path');
+
 const logger = require('./logger');
 
-const reportsDir = path.resolve(__dirname, '..', 'data', 'reports');
-const snapshotDir = path.resolve(__dirname, '..', 'data', 'snapshots');
-const states = path.resolve(__dirname, '..', 'data', 'states');
+const Cron = class Cron {
```

Comment:
Unnecessary `const` declaration

```suggestion
class Cron {
```